app/src/main/java/comp_sci_squad/com/github/url_irl/MainActivity.java

```diff
@@ -45,174 +49,44 @@
         ActivityCompat.OnRequestPermissionsResultCallback {
     /**
      * Extra Name Constants for the ListURLsActivity Intent
-     *
+     * <p>
```

Comment:
What is this

Reply:
I set auto clean up code and I guess it added this for better JavaDoc?